assets/js/components/FeatureTours.test.js

```diff
@@ -104,7 +104,7 @@ describe( 'FeatureTours', () => {
 		await waitForRegistry();
 
 		expect( observeMock ).toHaveBeenCalledWith(
-			container.querySelector( dashboardElementClassName )
+			container.querySelector( `#${ dashboardElementID }` )
```

Comment:
We could change this to avoid the string interpolation, although it works fine as it is :man_shrugging: 

```suggestion
			document.getElementById( dashboardElementID )
```

Reply:
It reads cleaner, but also seems a bit odd to reach for `document` directly rather than using the `container` returned by `render`. Ultimately, it's probably the same though.

Reply:
Yeah, I'd prefer to rely on exactly the returned variables from the render.